cluster/gce/config-default.sh

```diff
@@ -122,7 +122,7 @@ ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 # of fluentd running on a node, kubelet need to mark node on which
 # fluentd is not running as a manifest pod with appropriate label.
 # TODO(piosz): remove this in 1.8
-NODE_LABELS="${KUBE_NODE_LABELS:-beta.kubernetes.io/fluentd-ds-ready=true}"
+NODE_LABELS="${KUBE_NODE_LABELS:-beta.kubernetes.io/fluentd-ds-ready=true,beta.kubernetes.io/calico-ds-ready=true}"
```

Comment:
Why would this be `beta.kubernetes.io/calico-ds-ready=true` ? Shouldn't it be something like: `beta.calicoproject.org/ds-ready=true` ?

I guess my question comes down to 'who defined this as 'beta' and 'why does it belong in the kubernetes.io org?